modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -297,5 +298,29 @@ def breakout_date_components(date:)
     def build_application_expiration
       (Time.zone.now.to_date + 1.year).to_s
     end
+
+    # EVSS requires disability names to be less than 255 characters and cannot contain special characters.
+    # Rather than raise an exception to the user, massage the name into a valid state that EVSS will accept.
+    def massage_invalid_disability_names
+      disabilities = form_data.dig('disabilities')
+      invalid_characters = %r{[^a-zA-Z0-9\\\-'\.,\/\(\) ]}
+
+      disabilities.map do |disability|
+        name = disability['name']
+        name = truncate_disability_name(name: name) if name.length > 255
+        name = sanitize_disablity_name(name: name, regex: invalid_characters) if name.match?(invalid_characters)
```

Comment:
stylistic nitpick: I think the 2 helper methods are fine as inline calls (more readable and they're short calls that are clear as to what they're doing), but I'll leave that up to you.